lms/static/js/student_account/views/AccessView.js

```diff
@@ -85,9 +86,11 @@
             },
 
             render: function() {
-                $(this.el).html( _.template(this.tpl)({
+                var output = HtmlUtils.template(this.tpl)({
```

Comment:
I would lose this variable as it is only accessed once, and it will make it harder for the linter to see that the HTML is escaped.